camel/agents/chat_agent.py

```diff
@@ -134,6 +168,37 @@ def reset(self):
         self.terminated = False
         self.init_messages()
 
+    @property
+    def system_message(self) -> BaseMessage:
+        r"""Getter method for the property `system_message`
+
+        Returns:
+            BaseMessage: The system message of this agent.
+        """
+        return self._system_message
+
+    @system_message.setter
+    def system_message(self, value: BaseMessage):
+        r"""Setter method for the property `system_message`
+
+        Args:
+            value (BaseMessage): The value of the message to be set
+                as the new system message of this agent
+        """
+        self._system_message = value
+
+    @property
+    def func_enable(self) -> bool:
```

Comment:
Please use full words for function names. Here I suggest `is_function_enabled`. In English it would better be `are_functions_enabled`. I am okay with either of these.